content/learning-paths/cross-platform/simd-on-rust/simd-on-rust-part2.md

```diff
@@ -291,13 +291,13 @@ As you have seen, Rust has a very particular way to enable target features. In t
 unsafe fn sad_vec_asimd(a: &[u8], b: &[u8], w: usize, h: usize) -> u32 {
 ```
 
-Remember that `neon` support is implied with `dotprod` so there is no need to add it as well. However, this may not be the case for all extensions so you may have to add both, like this:
+Add support for both `neon` and `dotprod` target features as shown:
```

Comment:
I would still keep the comment above. 'dotprod' implies 'neon', but not vice versa.